src/api/api.c

```diff
@@ -413,6 +406,21 @@ int8_t z_encoding_to_string(const z_loaned_encoding_t *encoding, z_owned_string_
     return _Z_RES_OK;
 }
 
+int8_t z_slice_from_buf(z_owned_slice_t *slice, const uint8_t *data, size_t len) {
```

Comment:
function with similar name `z_bytes_from_str` takes ownersip of the buffer and accepts deleter and context. I think by analogy the `z_slice_wrap` should be named `z_slice_from_buf` and this `z_slice_from_buf` should be renamed to e.g. `z_slice_copy_from_buf` (of course we can't use word "serialize" here, it's reserved to payload).

I.e. all function names should follow the same logic:
- `z_xxx_from_yyy` takes ownership of it's parameter, providing deleter if necessary
- `z_xxx_copy_from_yyy` and `z_xxx_serialize_from_yyy` keeps parameter available to the caller

So all `z_xxx_wrap` frunctions should be renamed to `z_xxx_from_yyy`

Reply:
fixed